### PR TITLE
Doc minor issues

### PR DESCRIPTION
**Pull request by richardkapolnai-da**
### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/concepts/ledger-model/ledger-structure.rst [hunk 1]
```diff
@@ -153,7 +151,7 @@ In practice, multiple IOU contracts can exist between the same `obligor` and
 in this section, each contract only appears once, allowing us to drop the notion 
 of identifiers for simplicity reasons.
 
-A **transaction** is a list of actions. Thus, the consequences of
+A **transaction** is a non-empty list of actions. Thus, the consequences of
```
Comment (cocreature) [inline docs/source/concepts/ledger-model/ledger-structure.rst:154]: i don’t think we need this restriction in the ledger model, I’d omit it.

File: docs/source/concepts/ledger-model/ledger-structure.rst [hunk 2]
```diff
@@ -220,7 +218,7 @@ The following EBNF grammar summarizes the structure of commits and ledgers:
    Ledger   ::= Commit*
 
 A Daml ledger thus represents the full history of all actions taken by
-parties.\ [#ledger-vs-journal]_ Since the ledger is a sequence (of dependent actions), it induces an
+parties.\ [#ledger-vs-journal]_ Since the ledger has a sequence (of dependent actions), it induces an
```
Comment (cocreature) [inline docs/source/concepts/ledger-model/ledger-structure.rst:221]: I preferred the old phrasing here. From an abstract pov (and we are talking about the ledger model here). Saying the the whole ledger is a sequence of dependent actions seems fine.

File: docs/source/concepts/ledger-model/ledger-structure.rst (hunk 1 above)
Comment (richardkapolnai-da) [reply docs/source/concepts/ledger-model/ledger-structure.rst:154]: I think it should be either here, or a bit later:
"a proper subtransaction: a **non-empty** transaction obtained by removing at least one action, or"
the original definition maks an empty transaction a proper subtransaction, which is not I guess.
What do you think?

File: docs/source/concepts/ledger-model/ledger-structure.rst (hunk 2 above)
Comment (richardkapolnai-da) [reply docs/source/concepts/ledger-model/ledger-structure.rst:221]: yep, you're right, I got confused for a bit. reverting

File: docs/source/concepts/ledger-model/ledger-structure.rst (hunk 1 above)
Comment (cocreature) [reply docs/source/concepts/ledger-model/ledger-structure.rst:154]: What’s the issue with making an empty transaction a proper subtransaction?

Comment (bame-da) [reply docs/source/concepts/ledger-model/ledger-structure.rst:154]: All this terminology is borrowed from mathematics, where `sub-X` and `proper sub-X` have a fairly consistent meaning. And there the empty transaction would be considered a proper subtransaction of any non-empty transaction.

File: docs/source/concepts/ledger-model/ledger-structure.rst [hunk 3]
```diff
@@ -44,12 +44,10 @@ records the relations between the actions during the above acceptance.
 
 Formally, an **action** is one of the following:
 
-#. a **Create** action on a contract, which records the creation of the contract
```
Comment (bame-da) [inline docs/source/concepts/ledger-model/ledger-structure.rst:47]: This does sound odd, but is consistent in the Ledger Model. Actions are **on** contracts, and that includes the create node.

Comment (richardkapolnai-da) [reply docs/source/concepts/ledger-model/ledger-structure.rst:47]: Sorry, this have been confusing me for a long time along with the inexistent state. We have an infinite set of inexistent contracts in every model then?

File: docs/source/concepts/ledger-model/ledger-structure.rst (hunk 1 above)
Comment (richardkapolnai-da) [reply docs/source/concepts/ledger-model/ledger-structure.rst:154]: Then maybe the image is wrong?
"the image below shows all its proper subtransactions"
But the empty transaction is not on the image.
How about we add that too? An empty yellow box?

File: docs/source/concepts/ledger-model/ledger-structure.rst (hunk 3 above)
Comment (bame-da) [reply docs/source/concepts/ledger-model/ledger-structure.rst:47]: Yes, that is how the Ledger Model is set up.

File: docs/source/concepts/ledger-model/ledger-structure.rst (hunk 1 above)
Comment (bame-da) [reply docs/source/concepts/ledger-model/ledger-structure.rst:154]: Yeah, I'd be fine with relabelling the image to the "image below shows all its proper non-empty subtransactions".

File: docs/source/concepts/ledger-model/ledger-structure.rst (hunk 3 above)
Comment (richardkapolnai-da) [reply docs/source/concepts/ledger-model/ledger-structure.rst]: I reverted this.

**Review by bame-da — APPROVED** [review]
ty

**Pull request merged**